.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+go:
+  - 1.8.x
```

Comment:
Given that we're curling for `dep`, we could add `1.7` here as well.

Reply:
Yes we could, but we decided we don't want to officially support 1.7 so leaving it out.

Reply:
If we are dropping 1.7 then why don't we use dep from source?

Reply:
because like everything we want to take a dependency on actual release, why we would want a random version  that could break anytime?

Reply:
That's a fair argument. The answer to that question is because we get immediate adoption of improved bits.

To your point though, I think the question becomes: To what extent we trust the `dep` people to not make breaking changes.

Reply:
We can use a fixed version for now, however once dep goes GA we should re-evaluate.

Reply:
Seems like a good middle of the road option to me. Should we open an issue to remind us to take care of this?

Reply:
Will open one, but will put a strong recommend against on it :)

Reply:
https://github.com/Azure/azure-sdk-for-go/issues/1006